src/csharp/Grpc.Core/Internal/GrpcThreadPool.cs

```diff
@@ -176,10 +176,10 @@ private void RunHandlerLoop(CompletionQueueSafeHandle cq, IProfiler optionalProf
                     try
                     {
                         var callback = cq.CompletionRegistry.Extract(tag);
-                        // Use cached delegates to avoid unnecessary allocations
+                        queuedContinuationCounter.Increment();
                         if (!inlineHandlers)
```

Comment:
Realizing now, are we not setting `inlineHandlers` to true anywhere, including benchmarks?

Reply:
Yes, it's not being used in benchmarks because I thought it's also important to tests the performance with the "default" settings (right now the C# performance actually isn't much different with inlineHandlers==true because there are some other bottlenecks - i'm investigating).